wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -707,6 +707,23 @@ def admin_theme_classname(context):
     contrast_name = contrast_name.split("_")[0]
     return f"w-theme-{theme_name} w-density-{density_name} w-contrast-{contrast_name}"
 
+@register.simple_tag(takes_context=True)
+def admin_theme_user(context):
+    """
+    Retrives theme of current user
```

Comment:
This is inaccurate. It retrieves the **color scheme**, not the theme.

```suggestion
@register.simple_tag(takes_context=True)
def admin_theme_color_scheme(context):
    """
    Retrieves the color-scheme of the active theme for the current user.
```